staging/src/k8s.io/api/extensions/v1beta1/types.go

```diff
@@ -475,6 +487,9 @@ type DaemonSetCondition struct {
 
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:status-gen:introduced=1.1
```

Comment:
Will this show up in godoc?  I really want this in godoc and api doc...

Reply:
> Will this show up in godoc? I really want this in godoc and api doc...

godoc, no because it's in the comment section above like other generator tags

api doc, not yet, but maybe as a future step/verifier.  It *should* be possible to load openapi and diff against the metadata we generated.  This would provide the information to diff against, but I don't want this generator trying to mutate openapi. The metadata to the handler is more valuable at the moment.